package.json

```diff
@@ -53,6 +53,8 @@
   "activationEvents": [
     "onLanguage:puppet",
     "onLanguage:puppetfile",
+    "workspaceContains:**/*.pp",
+    "workspaceContains:**/metadata.json",
```

Comment:
Curious as to the reason for the permissive glob instead of just `metadata.json`

Reply:
Was thinking to catch situations like control repo, where the metadata or manifest is several levels deep